specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1469 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Batch Text Translation API",
+    "version": "v1.0-preview.1"
+  },
+  "host": "westus2.cognitiveservices.azure.com",
+  "basePath": "/translator/text/batch",
```

Comment:
Since this feature is essentially **document** batch translation (/translate also takes an batch of inputs as an array), it would be worth calling this out in the root name.  That way, instead of having /batch/batches, you'll have /document/batches.

Reply:
/translator/text/document